### ISSUE TITLE
Demo is not working on IE8

### ISSUE THREAD

**Issue by undeidentity** [danialfarid/ng-file-upload#753]
Is ng file upload still working on IE8?
I tried 3.2.3 and 3.1.0 either but all of these didn't work with same error

https://angular-file-upload.appspot.com/
Upload on form submit Test

Log 

FileAPI: 2.0.7
protocol: https:
doctype: [undefined] undefined undefined
FlashAPI.state: awaiting
(flash -> js).log: storeKey=408061644022050727353072930307296040403_2.0.7
(flash -> js).log: timeout=0
FlashAPI.state: ready
FlashAPI.event.mouseenter: [object Object]
(js -> flash).multiple: false
(flash -> js).log: parseCmd, command: multiple
(js -> flash).accept: art,bm,bmp,dwg,dxf,cbr,cbz,fif,fpx,gif,ico,iefs,jfif,jpe,jpeg,jpg,jps,jut,mcf,nap,nif,pbm,pcx,pgm,pict,pm,png,pnm,qif,qtif,ras,rast,rf,rp,svf,tga,tif,tiff,xbm,xpm,xwd,ART,BM,BMP,DWG,DXF,CBR,CBZ,FIF,FPX,GIF,ICO,IEFS,JFIF,JPE,JPEG,JPG,JPS,JUT,MCF,NAP,NIF,PBM,PCX,PGM,PICT,PM,PNG,PNM,QIF,QTIF,RAS,RAST,RF,RP,SVF,TGA,TIF,TIFF,XBM,XPM,XWD
(flash -> js).log: parseCmd, command: accept
FlashAPI.event.mouseDown: [object Object]
FlashAPI.event.select: [object Object]

Error: An undefined(unknown) error occurred.
(The error occurs on submit and the message may not accurate since i am not using windows which language is english)

I am using native IE8 on Windows7(vmware), not emulator.
I tried clearing cache.

thanks

*I tested IE8 on WindowsXP(vmware) few minutes ago, still get same error..

**Comment by undeidentity** [comment]
problem solved.
I've tested with modern.ie vmware image and it works.
But I still have no idea why ie8 on other vm images do not work..
I even downgraded my own windows7

**Comment by danialfarid** [comment]
So is this solved? Was it vm issue?

**Comment by undeidentity** [comment]
I don't know exactly but it's not only vm problem since the problem still occurs without vm. (windows7, ie8)

**Comment by undeidentity** [comment]
I have no idea why OS really matters. I have two VM Windows XP. One is downloaded on my own, the other one is downloaded from modern.ie. 
modern.ie works well, but the other one do not works. Both IE8 version and shockwave flash version are same.

**Comment by danialfarid** [comment]
Does this work? http://3-2-0-dot-angular-file-upload.appspot.com/

**Comment by undeidentity** [comment]
I've tested with VMWare Windows7 (downloaded from modern.ie) and My own Windows 7 ie8 without VM.
The former(vm) works, the latter still do not work with same problem.
Both IE8 version is 8.0.7601.17514, shockwave version is 17.
The special thing is, my own windows7 ie8 (which one has problem) browser mode falls into 'IE8 Compat View' and document falls into 'IE7 Standard' Mode while loading demo page. of course I've set Browser mode to IE8 and Document mode to IE8 Standard manually before test. (http://puu.sh/hXdOB/0547efac3e.png)
